openc3.sh

```diff
@@ -39,8 +39,17 @@ case $1 in
     # mapped as volume (-v) /openc3/local and container working directory (-w) also set to /openc3/local.
     # This allows tools running in the container to have a consistent path to the current working directory.
     # Run the command "ruby /openc3/bin/openc3cli" with all parameters starting at 2 since the first is 'openc3'
-    args=`echo $@ | { read _ args; echo $args; }`
-    docker run --rm -v `pwd`:/openc3/local -w /openc3/local openc3/openc3-base:$OPENC3_TAG ruby /openc3/bin/openc3cli $args
+    user=''
+    # Test for the --user=root (or whatever) as the second parameter
+    if [[ $2 == "--user"* ]]; then
+      user=$2
+      # Remove the first (openc3cli) and second (--user=root) parameters
+      shift; shift
+    else
+      # Remove the first (openc3cli) parameter
+      shift;
+    fi
```

Comment:
why is all this needed?

Reply:
I wanted to make the '--user=root' argument optional since it only seems to matter in some instances

Reply:
This only works with =root, with space root it won't work.

Reply:
Also, what about windows support?

Reply:
Probably just make a second keyword 'openc3cliroot' that adds the user root, rather than do weird parsing.  That will be easier to implement for windows too.